distributed/worker.py

```diff
@@ -3259,13 +3259,12 @@ async def _get_data():
 
 job_counter = [0]
 
-
 cache_loads = LRU(maxsize=100)
-
+enable_caching = dask.config.get("distributed.worker.cache", False)
```

Comment:
I can't find where this is defined. I would expect it to be in this file: https://github.com/dask/distributed/blob/master/distributed/distributed-schema.yaml#L228 and in here https://github.com/dask/distributed/blob/master/distributed/distributed.yaml#L58. Plus is should be `True` by default to preserve existing behavior.

Reply:
If we want to add this then it will have to have a more precise name.  Worker caching may refer to many different things other than this.